browser-test/src/trusted_intermediary.test.ts

```diff
@@ -668,6 +668,66 @@ test.describe('Trusted intermediaries', () => {
     await validateScreenshot(page, 'manage-ti-group-members-page')
   })
 
+  test('sort trusted intermediaries based on selection', async ({
+    page,
+    adminTiGroups,
+  }) => {
+    await loginAsAdmin(page)
+    // add "aaa" with 2 members
```

Comment:
Super nit: group aaa has 3 members

Reply:
Is there supposed to be a code update? Or is there really only 2 members? (I see foo, foo2, and foo3)